math/sigmoid.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <cmath>  // Untuk fungsi exp()
+using namespace std;
+
+/**
+ * @file sigmoid.cpp
+ * @brief Program ini menghitung fungsi sigmoid yang digunakan
+ * pada project Machine Learning dan Deep Learning. 
+ * Fungsi sigmoid akan menghasilkan nilai antara 0 hingga 1 
+ * berdasarkan input.
+ */
+double sigmoid(double x) {
+/**
+ * @brief Fungsi Sigmoid untuk menghitung nilai aktivasi
+ * yang berada di antara 0 hingga 1.
+ * @param x ini merupakan variabel input nilai.
+ * Nilai yang diberikan adalah tipe double dan hasilnya
+ * berupa nilai sigmoid yang juga dalam tipe double.
+ * @return Mengembalikan nilai sigmoid dari input x.
+ */
+    return 1.0 / (1.0 + exp(-x));  // Rumus sigmoid
+}
+
```

Comment:
```suggestion

/**
* @brief fungsi testing
* @return void
*/
void testing_sigmoid() {
	// untuk membandingkan hasil float
	const double toleransi = 1e-6;
	// contoh sederhana unittesting
	assert(fabs(sigmoid(0) - 0.5) < toleransi):
	std::cout << "testing berhasil" << std::endl;
}
```

contoh penggunaan unittesting untuk kasus fungsi sigmoid